quiz.html

```diff
@@ -34,6 +35,7 @@
 </head>
 
 <body>
+	<!-- instructor comments -->
 	<center>
```

Comment:
use margin 0 auto on wrapping container instead of <center> to separate style from markup